crates/jstz_cli/src/logs/trace.rs

```diff
@@ -34,8 +34,8 @@ pub async fn exec(
                 }
             }
             Err(err) => {
-                println!("Event source error: {}", err);
                 event_source.close();
+                eprintln!("Event source error, with an error: {}", err);
```

Comment:
Is this really better than the original?

Reply:
It should've been `Event source closed with an error: {}`. Will fix

Reply:
Fixed